App.js

```diff
@@ -1,20 +1,38 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { useFonts } from 'expo-font';
+import ProgressCard from './App/components/ProgressCard';
+
 
 export default function App() {
+  let [fontsLoaded] = useFonts({
+    'BrandonGrotesque-Regular': require('./App/assets/fonts/brandon-grotesque-regular.otf'),
+    'BrandonGrotesque-Bold': require('./App/assets/fonts/brandon-grotesque-bold.otf'),
+    'BrandonGrotesque-Light': require('./App/assets/fonts/brandon-grotesque-light.otf'),
+    'BrandonGrotesque-Medium': require('./App/assets/fonts/brandon-grotesque-medium.otf'),
+    'BrandonGrotesque-Black': require('./App/assets/fonts/brandon-grotesque-black.otf'),
+  });
+
+  if (!fontsLoaded) {
+    return null; 
+  }
+
   return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+      <View style={styles.borderContainer}>
```

Comment:
no need to add borderContiner

Reply:
Border container is for different purposes kindly look the figma there is a yellow shade border with container we can't apply that.